ARCs/arc-0069.md

```diff
@@ -31,9 +31,9 @@ The ASA parameters should follow the following conventions:
 * *Asset Name* (`an`): no restriction.
 * *Asset URL* (`au`): a URI pointing to digital media file. This URI:
     * **SHOULD** be persistent.
-    * **SHOULD** link to a file small enough to fetch quickly in a gallery view.
+    * **SHOULD** link to a file small enough to fetch quickly in a gallery view for files different from 3D Mesh. For 3D Mesh files it is **RECOMMENDED** that a client application downloads and renders them.

```

Comment:
I like the original wording on this line, it could apply to 3D meshes already

Reply:
It is true that the original wording could apply to 3d meshes. But it depends heavily on the purpose the mesh was made. A low poly could fit in a gallery.  A 3d mesh  in high definition made for historical simulations or architecture purpose could weigh up to GB.

Reply:
In this case it is not good. What if I wanted a high definition mesh NFT indicating the original binary blob in the IPFS hash? This especially makes sense when using NFT for Virtual Worlds / Metaverse